CMakeLists.txt

```diff
@@ -165,6 +167,7 @@ include_directories(
   ${EIGEN3_INCLUDE_DIRS}
   ${VTK_INCLUDE_DIRS}
   ${CMAKE_CURRENT_BINARY_DIR}
+  ${uima_INCLUDE_DIRS}
```

Comment:
this should not be needed it it is a ROS package...caktin_INCLUDE_DIRS should already contain all the header files from uima